pkg/psmdb/service.go

```diff
@@ -230,6 +241,11 @@ func GetAddr(m *api.PerconaServerMongoDB, pod, replset string) string {
 		":" + strconv.Itoa(int(m.Spec.Mongod.Net.Port))
 }
 
+// GetMCSAddr returns ReplicaSet pod address using MultiCluster FQDN
+func GetMCSAddr(m *api.PerconaServerMongoDB, pod string) string {
+	return fmt.Sprintf("%s.%s.%s:%d", pod, m.Namespace, m.Spec.MultiCluster.DNSSuffix, m.Spec.Mongod.Net.Port)
```

Comment:
`Spec.Mongod.Net.Port` will be deleted in #882, let's use the default port here too